### ISSUE TITLE
webcam stream not working, snapshot OK

### ISSUE THREAD

**Issue by Swat79** [OctoPrint/octoprint-docker#174]
I've got the problem, that I cant get the webcam stream to work.
Snapshot is working, FFMPEG says, path not exists.
Webcam is a C270 and was working outside docker properly. 
Any suggestions?

**Comment by graemedewe** [comment]
just discovered the same issue after I did pull and recreated my containers yesterday

**Comment by LongLiveCHIEF** [comment]
Let me know if the following commands work. If so, could be a bug related to the most recent release.


```
$ docker-compose exec octoprint apt-get install -y libv4l-dev 
$ docker-compose restart octoprint
```

**Comment by LongLiveCHIEF** [comment]
should be fixed after reverting #178
